core/src/main/scala/org/apache/spark/SecurityManager.scala

```diff
@@ -551,13 +553,10 @@ private[spark] class SecurityManager(
 
 private[spark] object SecurityManager {
 
-  val SPARK_AUTH_CONF: String = "spark.authenticate"
-  val SPARK_AUTH_SECRET_CONF: String = "spark.authenticate.secret"
+  val SPARK_AUTH_CONF = NETWORK_AUTH_ENABLED.key
+  val SPARK_AUTH_SECRET_CONF = "spark.authenticate.secret"
```

Comment:
I think we can also make this as a `ConfigEntry`.

Reply:
In a separate change maybe. There are too many references to this right now, it would be really noisy here.